src/document/generator.ml

```diff
@@ -21,12 +21,12 @@ open Types
 module O = Codefmt
 open O.Infix
 
+let tag tag t = O.span ~attr:tag t
+
 let label t =
   match t with
-  | Odoc_model.Lang.TypeExpr.Label s -> O.txt s
-  | Optional s -> O.txt "?" ++ O.txt s
-
-let tag tag t = O.span ~attr:tag t
+  | Odoc_model.Lang.TypeExpr.Label s -> tag "opt-arg" (O.txt s)
+  | Optional s -> tag "named-arg" (O.txt "?" ++ O.txt s)
```

Comment:
Please avoid inventing new terminology, I'd suggest `label` and `optlabel` as per [language specification](https://v2.ocaml.org/releases/5.0/htmlman/lex.html#sss:labelname).

Reply:
(Also you need to swap the names)

Reply:
Good points, thanks. I updated the PR!